tools/explorer/lib/ontology.py

```diff
@@ -42,7 +46,31 @@ def GetFieldsForTypeName(self,
     Returns:
             result_fields: a list of StandardField tuples
     """
-    pass
+    entity_type = self.universe.entity_type_universe.GetEntityType(
+        namespace,
+        entity_type_name
+    )
+    qualified_fields = entity_type.GetAllFields()
```

Comment:
standard_fields ?

Reply:
I shortened the code, so hopefully it makes a little more sense now.